lib/box-node-sdk.js

```diff
@@ -118,6 +118,67 @@ BoxSDKNode.collaborationRoles = BoxSDKNode.prototype.collaborationRoles = BoxCli
 BoxSDKNode.CURRENT_USER_ID = BoxSDKNode.prototype.CURRENT_USER_ID = BoxClient.prototype.CURRENT_USER_ID;
 
 /**
+ * Gets the BoxSDKNode instance by passing boxAppSettings json downloaded from the developer console.
+ *
+ * @param {object} appConfig boxAppSettings json is passed as parameter.
```

Comment:
A couple issues with this JSDoc:
1. Non-primitive types should de documented capitalized, e.g. `{Object}`
1. The parameter description is a bit confusing; this takes an Object, not a JSON string
1. No need to say it's passed as a parameter, since this is a `@param` tag after all